js/app.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener(`DOMContentLoaded`, () =>{
+
+	const form = document.querySelector(`#registrar`);
+	const formInput = form.querySelector(`input`);
+	const invitedUl = document.querySelector(`#invitedList`);
+	const hideNotResponded = document.querySelector(`.main input`);
+
+	function newElement(elementParent, element) {
+		const name = document.createElement(element);
```

Comment:
Strasznie duże wcięcia, ustaw sobie taby na 2 spacje :)

Reply:
w atome wygląda normalnie ;p

Reply:
Miks w webstormie ctrl+alt+L to jest auto-stylowanie może w atomie też? :)